assets/src/js/admin/form-template/Sequoia/edit-form.js

```diff
@@ -0,0 +1,58 @@
+/* globals Give, jQuery */
+( function( $ ) {
+	/**
+	 * Return whether or not recurring addon enabled or not.
```

Comment:
Clean up the wording:

> Return whether or not the recurring addon is enabled or not.

Reply:
fixed: https://github.com/impress-org/givewp/compare/d41048139732...923867334974